__tests__/src/containers/DataPackageViewContainer.test.js

```diff
@@ -73,7 +65,7 @@ const mockData = [
 
 describe("Datapackage View Container", () => {
 
-  const wrapper = shallow(<DataPackageViewContainer />);
+  const wrapper = shallow(<DataPackageView publisherName="publisher" packageName="pack" metadata={{readme: ''}}/>);
```

Comment:
i really do not know why travis is not working. It built twice after that it stopped woking. May be it was only running the branch i created. I have to look at that.